libdnf/transaction/MergedTransaction.cpp

```diff
@@ -308,12 +310,14 @@ MergedTransaction::resolveErase(ItemPairMap &itemPairMap,
     /*
      * The original item has been removed - it has to be installed now unless the rpmdb
      *  has changed. Resolve the difference between packages and mark it as Upgrade,
-     *  Reinstall or Downgrade
+     *  Downgrade or drop it from the transaction
```

Comment:
Just a little thing.
On line 267 you replace the word `drop` with `remove` and use `drop` here again.